src/components/molecules/OakHandDrawnHR/OakHandDrawnHR.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled from "styled-components";
+
+import { OakFlex } from "../../atoms";
+import {
+  InternalStyledSvg,
+  InternalStyledSvgProps,
+} from "../../atoms/InternalStyledSvg";
+
+import { SizeStyleProps } from "@/styles/utils/sizeStyle";
+import { SpacingStyleProps } from "@/styles/utils/spacingStyle";
+
+const StyledOakFlex = styled(OakFlex)``;
+
+export type OakHandDrawnHRProps = {
+  fill?: InternalStyledSvgProps["$fill"];
+  stroke?: InternalStyledSvgProps["$stroke"];
+  strokeWidth?: InternalStyledSvgProps["$strokeWidth"];
+} & SpacingStyleProps &
+  SizeStyleProps;
+
+/**
+ * A Drawn HR svg inside a flex container
+ *
+ * An optional `stroke` and `strokeWidth` can be applied to give the background a border
+ */
+export const OakHandDrawnHR = (props: OakHandDrawnHRProps) => {
+  const { fill, stroke, strokeWidth, ...flexProps } = props;
+
+  return (
+    <StyledOakFlex {...flexProps}>
+      <InternalStyledSvg
```

Comment:
I wonder if this could live in a separate file somewhere . Either in this component folder or perhaps we start an assets folder for svgs ?